modules/ROOT/pages/salesforce-to-twilio.adoc

```diff
@@ -0,0 +1,454 @@
+```
+select * from products where category = "Enterprise" and inventory > 1 limit 1
+```
+.. Close the Database card.
+.. Add a Database card after the *Small Business* subcard, and select the *Select* operation.
++
+[NOTE]
+====
+Notice that you do not have to configure the database driver again. That’s because, when you configured it earlier, Flow Designer saved the configuration. If you look under the heading “Reusable Configurations” in the Projects sidebar, you’ll see an entry labelled “Database”.
+====
+.. Add this query to this Database card:
++
+```
+select * from products where category = "Small Business" and inventory > 1 limit 1
+```
+. Set up the default option for when the size of the deal is $50,000 or less. In this case, you do not need to query the database.
+.. Add a Set Payload card after the Default subcard. You use the Set Payload card here so that, if the Choice router block routes to this default option, the flow simply adds the required SKU and associated information.
+.. In the open Set Payload card, click the *f(x)* button to enter script mode in the *Value* field.
+.. Paste the following JSON data into the field:
++
+```
+[
+ {
+   "productID": "CK111274",
+   "category": "Retail",
+   "SKU": "CKS0",
+   "productName": "Old School Classic Kicks",
+   "inventory": 9999
+ }
+]
+```
+
+The flow should now look like this:
+
+image::flow-with-complete-choice.png[]
+
+== Change the format of the information before passing it to the NTO Order API
+
+By this point, the flow has obtained information about the opportunity from Salesforce and about the product from the product database. Before using this information to place an order with the NTO Order API, the flow needs to format it in the way that the API is expecting.
+
+. Click the plus sign to the right of the Choice router block and add a Transform card.
+. In the *Input* pane, specify input data types.
+.. Click the three dots next to `payload`.
+.. Select *Set*.
+.. Select the `Product Information` data type that you created earlier.
+.. Click the three dots next to `OppDetails` under `vars` and set the data type to `Salesforce Opportunity`.
+. In the *Output* pane, set the data type for `payload` to `Order Information`.
++
+*Result:* The Transform card should look like this:
++
+image::transform-first-after-input-output.png[]
+. Drag and drop elements from the left side to right side according to the mappings below:
++
+[%header,cols=2*]
+|===
+|This Element on the Left Side
+|Maps to This Element on the Right Side
+
+|`productID`
+|`productID`
+
+|`category`
+|`category`
+
+|`SKU`
+|`SKU`
+
+|`productName`
+|`productName`
+
+|`inventory`
+|`inventory`
+
+|`Amount`
+|`OrderAmount`
+
+|`Id`
+|`OppId`
+
+|`CloseDate`
+|`orderDate`
+
+|===
+
+If you see the message `Output media type differs from expected one`, click the `Fix it` link to resolve the problem.
+
+When you are done changing the format, the Transform card looks like this:
+
+image::transform-first-final.png[]
```

Comment:
@rddheath @jimiljojo can we agree that we will update this to start off using auto-mapper suggestions and then manually adjust what is left? (Once we fix automapper :) )

Reply:
Just let us know when the automapper is ready.